apps/api/src/app/workflows-v2/mappers/notification-template-mapper.ts

```diff
@@ -72,7 +72,7 @@ export function toWorkflowsMinifiedDtos(templates: NotificationTemplateEntity[])
 }
 
 function toStepResponseDto(persistedStep: NotificationStepEntity): StepResponseDto {
-  const stepName = persistedStep.name || 'Missing Name';
+  const stepName = persistedStep.name || 'Missing Name || UPDATE IMMEDIATELY';
```

Comment:
This needs to be here, either that or an exception in case of missing name, the entity allows missing names, but our API no more @SokratisVidros

Reply:
The entity shouldn't allow for missing names, and the API should return a 422 or 400 error if the name is missing for a step.

Reply:
I'm resolving the comment, we are discussing it offfline